.github/workflows/deploy-to-gcp.yaml

```
@@ -32,13 +32,23 @@
         with:
           version: 9
 
+      - name: Check for Terraform changes
+        id: check_changes
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:37: Double quote to prevent globbing and word splitting

(shellcheck)